tools/cldr-code/src/main/java/org/unicode/cldr/util/UnitConverter.java

```diff
@@ -365,15 +363,15 @@ public UnitIterator iterator() {
 
     public UnitConverter(RationalParser rationalParser, Validity validity) {
         this.rationalParser = rationalParser;
-        // we need to pass in the validity so it is for the same CLDR version as the converter
-        Set<String> VALID_UNITS = validity.getStatusToCodes(LstrType.unit).get(Status.regular);
-        Map<String,String> _SHORT_TO_LONG_ID = new LinkedHashMap<>();
-        for (String longUnit : VALID_UNITS) {
-            int dashPos = longUnit.indexOf('-');
-            String coreUnit = longUnit.substring(dashPos+1);
-            _SHORT_TO_LONG_ID.put(coreUnit, longUnit);
-        }
-        SHORT_TO_LONG_ID = ImmutableBiMap.copyOf(_SHORT_TO_LONG_ID);
+//        // we need to pass in the validity so it is for the same CLDR version as the converter
+//        Set<String> VALID_UNITS = validity.getStatusToCodes(LstrType.unit).get(Status.regular);
+//        Map<String,String> _SHORT_TO_LONG_ID = new LinkedHashMap<>();
+//        for (String longUnit : VALID_UNITS) {
+//            int dashPos = longUnit.indexOf('-');
+//            String coreUnit = longUnit.substring(dashPos+1);
+//            _SHORT_TO_LONG_ID.put(coreUnit, longUnit);
+//        }
+//        SHORT_TO_LONG_ID = ImmutableBiMap.copyOf(_SHORT_TO_LONG_ID);
```

Comment:
```suggestion
```

@btangmu has got me on the deletionist bandwagon… this code can't be used anymore due to the .inverse(). could be fixed in a follow up PR.
tom's got